app.js

```diff
@@ -24,7 +30,7 @@ app.use((req, res, next) => {
 // error handling
 
 app.use((err, req, res, next) => {
-  if (err.code === "22P02") {
+  if (err.code) {
     err.message = "Bad request";
```

Comment:
`err.code` will sometimes yield a code that would be a `404` error, so I would advise checking for the specific code